products/jslbre/features/all_sites.feature

```diff
@@ -0,0 +1 @@
+/Users/kmakitan/Documents/behat/linky_clicky/includes/features/all_sites.feature
```

Comment:
This link won't work on my machine because I'm not running in your account. Please create this link to be relative only to the project.